Tests fail to compile: FindThreads only works if either C or CXX language is enabled

```
CMake Error at /usr/local/share/cmake/Modules/FindThreads.cmake:66 (message):
  FindThreads only works if either C or CXX language is enabled
Call Stack (most recent call first):
  CMakeLists.txt:13 (find_package)
```

Version: 0.24.0
cmake-3.21.4
FreeBSD 13

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -8,7 +8,7 @@
 ############################################################################
 
 cmake_minimum_required(VERSION 3.1)
-project(xtensor)
+project(xtensor CXX)
 
 set(XTENSOR_INCLUDE_DIR ${CMAKE_CURRENT_SOURCE_DIR}/include)
 

--- a/test/CMakeLists.txt
+++ b/test/CMakeLists.txt
@@ -13,7 +13,7 @@ find_package(doctest            REQUIRED)
 find_package(Threads)
 
 if (CMAKE_CURRENT_SOURCE_DIR STREQUAL CMAKE_SOURCE_DIR)
-    project(xtensor-test)
+    project(xtensor-test CXX)
 
     enable_testing()
 

